blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/UserImpl.java

```diff
@@ -6,6 +6,8 @@
 import hudson.model.User;
 import hudson.tasks.Mailer;
 import hudson.tasks.UserAvatarResolver;
+import hudson.util.HttpResponses;
```

Comment:
@vivek this was the key... *not* the stapler version, but the `hudson.util` version has all those JSON utility methods...